### PR TITLE
feat(playwright): add pipeline code to playwright plugin

### PR DESCRIPTION
**Pull request by nandorszentpeteri**

### CODE REVIEW

**Review by johnmckenna — APPROVED** [review]

File: packages/playwright/src/generators/init/utils/update-targets.ts [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+            const updatedProjectJson = { ...projectJson };
+
+            if (!projectJson.targets['e2e-docker']) {
+                updatedProjectJson.targets['e2e-docker'] = {
+                    executor: 'nx:run-commands',
+                    options: {
+                        commands: [
+                            {
+                                command:
+                                    'docker run -v $PWD:/project -w /project --rm --ipc=host mcr.microsoft.com/playwright:v1.30.0-jammy npx nx run next-custom-server-e2e:e2e',
```
Comment (johnmckenna) [inline packages/playwright/src/generators/init/utils/update-targets.ts:43]: Does this need changed to a project name variable?

**Review by MetecanAydin — APPROVED** [review]

File: packages/playwright/src/generators/init/utils/update-targets.ts (hunk 1 above)
Comment (nandorszentpeteri) [reply packages/playwright/src/generators/init/utils/update-targets.ts:43]: will do

Comment (Danphillipz) [inline packages/playwright/src/generators/init/utils/update-targets.ts:43]: Is there any way of having the playwright docker image use the correct version automatically? In this approach we will have a maintenance issue for the plugin, whereby we have to update this image version within our generator. Users will also have to manually update this within their project files

File: packages/playwright/src/generators/init/utils/update-targets.ts [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+                                forwardAllArgs: false,
+                            },
+                        ],
+                    },
+                    configurations: {
+                        updatesnapshots: {
+                            args: '--extra=--update-snapshots',
```
Comment (Danphillipz) [inline packages/playwright/src/generators/init/utils/update-targets.ts:50]: Could we also get an additional arg?
--grep @visual-regression
When updating snapshots we wouldn't want to run the entire test suite, rather limit it to only the tests which actual perform visual regression

**Pull request merged**